### ISSUE TITLE
Experimental Settings

### ISSUE THREAD

**Issue by nttmac** [thunlp/RE-Context-or-Names#5]
Hi, thank you for sharing the code! 
I have read your great paper and have some questions about experimental settings in the pilot experiment. 

> There are five different experimental settings (C+M, C+T, Only C, Only M, and Only T), did you train five models with various input or only train the model using C+M training data and test it under these five settings? 

Thanks for your help.

**Comment by h-peng17** [comment]
I train the model under the five different settings and test the model under the corresponding setting. We shall keep the settings consistant on train set and test set.

**Comment by nttmac** [comment]
> I train the model under the five different settings and test the model under the corresponding setting. We shall keep the settings consistant on train set and test set.

Thank you very much! Have you uploaded the code about these five settings? I find only one RE model in "model.py" file.

**Comment by h-peng17** [comment]
The five settings are only different in the input/encoding fromat. I have implement the five different input setting in dataset.py. You can change the --mode para to set the setting.

**Comment by nttmac** [comment]
> The five settings are only different in the input/encoding fromat. I have implement the five different input setting in dataset.py. You can change the --mode para to set the setting.

Got! Appreciate your attention!
